src/accelerate/commands/launch.py

```diff
@@ -257,6 +257,12 @@ def simple_launcher(args):
 
     current_env = os.environ.copy()
     current_env["USE_CPU"] = str(args.cpu)
+    if args.num_machines > 1:
+        current_env["MASTER_ADDR"] = args.main_process_ip
+        current_env["MASTER_PORT"] = str(args.main_process_port)
+    elif args.num_processes > 1:
+        current_env["MASTER_ADDR"] = "127.0.0.1"
+        current_env["MASTER_PORT"] = "29500"
```

Comment:
If they are needed, I would rather use the ones set par the user than hardcoding a default, which may create hard-to-debug issues for users.

Reply:
hi, sgugger, now in the accelerate config, if num_machines == 1, the master ip and port is not set. and if num_process > 1 and DDP is used in one machine, we still need the master ip and port

Reply:
done, thanks for the suggestion